backend/src/modules/email-sending/core/application/command/SendEmailCommandHandler.ts

```diff
@@ -0,0 +1,19 @@
+import { CommandHandler } from '../../../../../shared/core/application/command/CommandHandler';
+import { SendEmail } from './SendEmail';
+import { CommandResult } from '../../../../../shared/core/application/command/CommandResult';
+import { EmailSender } from '../EmailSender';
+
+export class SendEmailCommandHandler implements CommandHandler<SendEmail> {
+  constructor(private readonly emailSender: EmailSender) {}
+
+  async execute(command: SendEmail): Promise<CommandResult> {
+    await this.emailSender.sendAnEmail({
+      from: 'TourDeFoos <TourDeFoos@gmail.com>',
```

Comment:
Dobrze jakby to też było parametryzowane jakoś. Chyba w EmailSenderze nie? Bo będziemy wysyłać z tego samego adresu, a reszta będzie się różnić. A jakby zmieniać z gmaila na coś innego to to już jest nieaktualne.

Reply:
Chyba trochę nie rozumiem (ale już też trochę zasypiam) ... przecież wywołanie komendy SendEmail powoduje wysłanie maila, a w wywołaniu tejże komendy  podaje się emailAddress, temat i zawartość maila. To po co jakieś pola, czy coś innego w interfejsie EmailSender?

Reply:
W NodeMailerEmailSender mam host, tak? To jak zmienie host np. na WP / o2 czy cokolwiek innego. No to już nie wysyłam z TourDeFoos@gmail.com - to też powinno być konfigurowalne w NodeMailerSender i juz nie ma po co przekazywać w sendAnEmail.

Reply:
Ale przecież przed wysłaniem maila muszę wywołać metodę NodeMailer.createTransport gdzie podaję email, host i port... jak te pola wyrzucę do interfejsu EmailSender to i tak muszę im przypisać jakieś wartości w NodeMailerEmailSender... 
Jeżeli chcemy zmienić serwis wysyłania mail to i tak byśmy musieli zrobić nowy obiekt typu NodeMailerEmailSender. 

Na razie na sztywno jest wpisany mail, z którego wysyłamy  - to faktycznie można zmienić .... <---- to po prostu o pole skąd wysyłany jest mail, Ci chodziło, taaak?

Reply:
https://github.com/nowakprojects/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments/pull/37/commits/0a482065413de1a7ef876ceb0983c21032adc1c0
to miałeś na myśli ? 
@nowakprojects

Reply:
Dokładnie! :)